data_model/src/lib.rs

```diff
@@ -458,8 +426,7 @@ impl Value {
         match self {
             U32(_) | U128(_) | Id(_) | PublicKey(_) | Bool(_) | Parameter(_) | Identifiable(_)
             | String(_) | Name(_) | Fixed(_) | TransactionValue(_) | PermissionToken(_)
-            | Hash(_) => 1_usize,
-            Block(v) => v.nested_len() + 1_usize,
+            | Hash(_) | Block(_) => 1_usize,
```

Comment:
So a block is a sing,e entity, rather than a collection of entitites. This was an extremely subtle mistake. Good catch.

Reply:
yeah, I remember commenting on the PR but I wasn't certain at the time what's correct